src/components/form/index.tsx

```diff
@@ -24,18 +37,24 @@ export class SmoothlyForm implements Clearable, Submittable, Editable {
 	@Prop() prevent = true
 	@Prop({ mutable: true }) changed = false
 	@State() processing?: Promise<boolean>
+	@Event() smoothlyFormDisable: EventEmitter<(disabled: boolean) => void>
 	@Event() smoothlyFormInput: EventEmitter<Data>
 	@Event() smoothlyFormSubmit: EventEmitter<Submit>
 	@Event() smoothlyFormReset: EventEmitter<void>
 	@Event() smoothlyFormEdit: EventEmitter<boolean>
 	@Event() smoothlyFormClear: EventEmitter<void>
 	@Event() notice: EventEmitter<Notice>
+	private binary = false
```

Comment:
-> "mime"/"mimeType": "json" | "formdata" = json
or something instead i think

Reply:
Will change. The idea was to only chose formdata if the form contains binary data.

Reply:
Settled for `contentType` since that is the real header name. Mime type would make sense for JSON but not for formdata. it is multipart and can have multiple mime data types.

Reply:
\+